### DISCUSSION TITLE
How to stop kubelet service running in an RKE kubernetes node?

### DISCUSSION THREAD

**Discussion by LittleTodd** [rancher/rke2#5625] [Q&A] [answered]
It looks like command _systemctl stop rke2-server.service_ would stop all containerd processes like rke2, kubelet, kube-proxy, and containerd processes, but how can I just stop kubelet.service, thanks!

**Comment by brandond** [comment, marked as answer]
There is only one service. Stopping RKE2 stops the supervisor process, kubelet, and containerd. Everything else (all the pods), including kube-proxy, remain running. There is no way to stop only part of RKE2.

**Reply by LittleTodd** [reply]
Thanks Brad, will try simulating Node "NotReady" status by stopping RKE2.
